app/views/admin/index.html.erb

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is the reason to have this file with any content?

Reply:
@JuanequeX for redirect to one file when de admin login successfully, and not made conflict when the teammate with that issue does his merge

Reply:
deleted file